common-codegen/src/main/java/io/vrap/rmf/codegen/common/generator/extensions/StringExtension.java

```diff
@@ -0,0 +1,37 @@
+package io.vrap.rmf.codegen.common.generator.extensions;
+
+import com.google.common.base.CaseFormat;
+import com.google.common.base.Converter;
+import io.vrap.rmf.codegen.common.processor.annotations.ExtensionMethod;
+import io.vrap.rmf.codegen.common.processor.annotations.ModelExtension;
+import org.apache.commons.lang3.StringUtils;
+
+import java.text.CharacterIterator;
+import java.text.StringCharacterIterator;
+
+@ModelExtension(extend = String.class)
+public class StringExtension {
+
+    private static final Converter<String, String> classNameMapper = CaseFormat.LOWER_HYPHEN.converterTo(CaseFormat.UPPER_CAMEL);
```

Comment:
Take a look at io.vrap.rmf.raml.model.util.StringCaseFormat which takes any cased input and converts it to the target format.